sw/device/tests/usbdev_test.c

```diff
@@ -89,7 +89,7 @@ bool test_main(void) {
   // Call `usbdev_init` here so that DPI will not start until the
   // simulation has finished all of the printing, which takes a while
   // if `--trace` was passed in.
-  usbdev_init(&usbdev);
+  usbdev_init(&usbdev,false, false, false);
```

Comment:
Consider writing `/*pinflip=*/ false` and similar in such function calls, to make which parameter is which clearer.

Reply:
Done